### PR TITLE
Added Text to voice converter

### PR DESCRIPTION
**Pull request by Aashi19234**
# Pull Request for WebMasterLog 💡

## Issue Title: Text to voice converter using Vanilla JS

- **Name of the project** : Text to voice converter
- **Category:** : Vanilla JS
- **Name:** : Aastha Jain
- **GitHub ID:** : https://github.com/Aashi19234
- **Idenitfy yourself:** JWOC'24 Contributor

Closes: #69 

### Describe the add-ons or changes you've made 📃

Give a clear description of what have you added or modifications made

## Type of change ☑️

- [X] New feature (non-breaking change which adds functionality)

## How Has This Been Tested? ⚙️

I have tested the program by running it on my local system by entering the text and it returned a computed voice,

## Checklist: ☑️

- [X] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [X] This PR does not contain plagiarized content.
- [X] I have performed a self-review of my own code.
- [X] I have commented my code, particularly wherever it was hard to understand.
- [X] I have made corresponding changes to the documentation.
- [X] My changes generate no new warnings.
- [X] I have added things that prove my fix is effective or that my feature works.
- [X] Any dependent changes have been merged and published in downstream modules.

## Screenshots 📷

https://github.com/Avdhesh-Varshney/WebMasterLog/assets/140010647/c1090740-b206-4f85-8bd8-942454d94d97

## Note to reviewers 📄
Please check this PR and if you want any changes please let me know.

### CODE REVIEW

File: Vanilla-JS-Projects/Intermediate/Texttovoice/Screenshot (83).png
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Intermediate/Texttovoice/Screenshot (83).png:1]: Screenshot file name is same as that of directory name. And also change the directory name according to `Kebab-Case`.
- Directory name: `Text-to-Voice`
- Screenshot name: `Text-to-Voice.webp` as mentioned in Contribution guidelines.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@Aashi19234 Convert your png file into webp file format.
Again commit these changes and mentioned to me.

**Comment by Aashi19234** [comment]
@Avdhesh-Varshney  I have made the required changes.

**Review by Avdhesh-Varshney — APPROVED** [review]
PR Approved. 🎉

**Pull request merged**